buildSrc/build.gradle.kts

```diff
@@ -14,7 +14,7 @@ repositories {
 object PluginVersions {
     const val GRADLE_ANDROID = "7.0.0"
     const val GRADLE_VERSIONS = "0.33.0"
-    const val KOTLIN = "1.5.21"
+    const val KOTLIN = "1.5.10"
```

Comment:
You don't need to lower kotlin version if you use compose `1.0.1`

Reply:
Ah! That's great. I'll update the version.